src/classes/Launch/Webhook.ts

```diff
@@ -68,7 +56,20 @@ export class Webhook {
           throw error(e);
         }
       })
-      .listen(config.port || 80);
+      .listen(this.webhookConfig.port || 80);
+  }
+
+  private registerWebhookIfNecessary(): Promise<void> {
+    return this.api
+      .getWebhookInfo()
+      .then((webhookInfo) => {
+        // An empty webhook URL indicates that the webhook is not registered.
+        if (webhookInfo.url.length !== 0) return;
+        if (this.runConfig.logging) info('Re-registering webhook...');
+
+        return this.api.setWebhook(this.webhookConfig);
+      })
+      .then(() => {});
```

Comment:
Why have this `.then(() => {})`?
No need to make something extra on the call stack

Reply:
Lots of `Api` methods return boolean for some reason, and this coerces it to `void`.